spec/unit/installer/xcode/target_validator_spec.rb

```diff
@@ -42,8 +42,8 @@ def create_validator(sandbox, podfile, lockfile)
             config.repos_dir = fixture_path + 'spec-repos'
             podfile = Pod::Podfile.new do
               platform :ios, '8.0'
-              install! 'cocoapods', :integrate_targets => false
-              project 'SampleProject/SampleProject'
+              install! 'cocoapods', :integrate_targets => true
```

Comment:
all of these tests were "wrong" as in they were never taking into consideration `:integrate_targets => false` above and the project was never properly loaded. This is now fixed.